components/admin_console/workspace-optimization/dashboard.tsx

```diff
@@ -325,41 +333,11 @@ const WorkspaceOptimizationDashboard = (props: Props) => {
 
             // chips will only be displayed for info aka Success, warning and error aka Problems
             if (item.status && item.status !== 'none' && item.status !== 'ok') {
-                chipsInfo[item.status] += 1;
+                accordionDataChips[item.status] += 1;
+                overallScoreChipsData[item.status] += 1;
             }
         });
-        const chipsList = Object.entries(chipsInfo).map(([chipKey, count]) => {
-            if (count === 0) {
-                return false;
-            }
-            let id;
-            let defaultMessage;
-
-            switch (chipKey) {
-            case 'info':
-                id = 'admin.reporting.workspace_optimization.suggestions';
-                defaultMessage = 'Suggestions';
-                break;
-            case 'warning':
-                id = 'admin.reporting.workspace_optimization.warnings';
-                defaultMessage = 'Warnings';
-                break;
-            case 'error':
-            default:
-                id = 'admin.reporting.workspace_optimization.problems';
-                defaultMessage = 'Problems';
-                break;
-            }
-
-            return (
-                <Chip
-                    key={chipKey}
-                    id={id}
-                    defaultMessage={`${defaultMessage}: ${count}`}
-                    className={chipKey}
-                />
-            );
-        });
+        const chipsList = (<ChipsList chipsData={accordionDataChips}/>);
```

Comment:
no need to create a variable for that ... you can just assign that directly in the return:

```typescript
return {
    title,
    description,
    icon,
    items,
    extraContent: <ChipsList chipsData={accordionDataChips}/>,
};
```

Reply:
ok